### PR TITLE
Fixed a Regression Bug

### PR DESCRIPTION
**Pull request by ajinabraham**
### What was a problem?

```
https://github.com/MobSF/Mobile-Security-Framework-MobSF/issues/662
```

### How this PR fixes the problem?

```
Use CapFuzz root CA
Also Code QA
```

### Check lists (check `x` in `[ ]` of list items)

- [x] Run MobSF unit tests
- [x] Tested Working on Linux, Mac, and Windows
- [x] Coding style (indentation, etc)

### Additional Comments (if any)

```
DESCRIBE HERE
```

### CODE REVIEW

**Comment by ajinabraham** [comment]
This PR fixes #662

File: scripts/mobsfy.py [hunk 1]
```diff
@@ -36,100 +47,129 @@ def getADB():
...
+        print("\n[INFO] Installing Xposed Framework")
+        execute_cmd([get_adb(), "-s", adbconnect, "install", "-r", xposed])
+        print("\n[INFO] Installing Droidmon API Analyzer")
+        execute_cmd([get_adb(), "-s", adbconnect, "install", "-r", droidmon])
+        print("\n[INFO] Installing JustTrustMe")
+        execute_cmd([get_adb(), "-s", adbconnect, "install", "-r", justrustme])
+        print("\n[INFO] Installing RootCloak")
+        execute_cmd([get_adb(), "-s", adbconnect, "install", "-r", rootcloak])
+
+        if vm_or_ip == "1":
```
[the remark this answers is not in the document]
Comment (ajinabraham) [reply scripts/mobsfy.py]: @magaofei any advantage?
We read it from user: `vm_or_ip = input("Choose\n 1. VM\n 2. Device\nEnter your choice: ")`

[the remark this answers is not in the document]
Comment (magaofei) [reply scripts/mobsfy.py]: Reduce the magic value in the code, the variable name is clearer

[the remark this answers is not in the document]
Comment (ajinabraham) [reply scripts/mobsfy.py]: fair enough.

[the remark this answers is not in the document]
Comment (ajinabraham) [reply scripts/mobsfy.py]: @magaofei made the changes

**Pull request merged**